widgets/alternate/plus.svg

```diff
@@ -22,7 +22,7 @@
          rdf:about="">
         <dc:format>image/svg+xml</dc:format>
         <dc:type
-           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
```

Comment:
Please remove any changes to svg files, they are not relevant. Not only in this hunk, and all svg files changed.